Wikipedia/UI-V5/WMFTableOfContentsCell.swift

```diff
@@ -0,0 +1,167 @@
+
+import UIKit
+
+// MARK: - Cell
+public class WMFTableOfContentsCell: UITableViewCell {
+
+    @IBOutlet var sectionSelectionBackground: UIView!
+    @IBOutlet var sectionTitle: UILabel!
+    @IBOutlet var selectedSectionIndicator: UIView!
+    @IBOutlet var indentationConstraint: NSLayoutConstraint!
+    @IBOutlet var leadSectionBorder: UIView!
+    @IBOutlet var sectionBorder: UIView!
+    
+    // MARK: - Init
+    public required init?(coder aDecoder: NSCoder) {
+        section = nil
+        super.init(coder: aDecoder);
+        self.indentationWidth = 10
+    }
+    
+    // MARK: - UIView
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+        self.selectedSectionIndicator.alpha = 0.0
+        self.sectionSelectionBackground.backgroundColor = UIColor.wmf_tableOfContentsSelectionBackgroundColor()
+        self.sectionSelectionBackground.alpha = 0.0
+        self.selectionStyle = .None
+    }
+    
+    // MARK: - Accessors
+    public var section: MWKSection?{
+        didSet(section) {
+            self.sectionTitle.text = WMFTableOfContentsCell.titleText(self.section)
+            self.sectionTitle.font = WMFTableOfContentsCell.titleFont(self.section)
+            self.sectionTitle.textColor = WMFTableOfContentsCell.textColor(self.section)
+            self.leadSectionBorder.hidden = !WMFTableOfContentsCell.leadSectionBorderEnabled(self.section)
+            self.sectionBorder.hidden = !WMFTableOfContentsCell.topBorderEnabled(self.section)
+            self.indentationConstraint.constant = WMFTableOfContentsCell.indentationWidth(self.section)
+            self.setNeedsUpdateConstraints()
+        }
+    }
+    
+    public func setSectionSelected(selected: Bool, animated: Bool) {
+        UIView.animateWithDuration(animated ? 0.3 : 0.0) { () -> Void in
+            if(selected){
+                self.sectionSelectionBackground.alpha = 1.0
+            }else{
+                self.sectionSelectionBackground.alpha = 0.0
+            }
+        }
+    }
+    
+    // MARK: - UITableVIewCell
+    public class func reuseIdentifier() -> String{
+        return self.wmf_nibName()
+    }
+    
+    public override func prepareForReuse() {
+        super.prepareForReuse()
+        self.section = nil
+        self.leadSectionBorder.hidden = true
+        self.sectionBorder.hidden = true
+        self.setSelected(false, animated: false)
+        self.sectionSelectionBackground.alpha = 0.0
+    }
+    
+    public override func setSelected(selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        if(selected){
+            //HACK: I don't know why I have to set the color here, but I do. Something is setting it to clear background color
+            self.selectedSectionIndicator.backgroundColor = UIColor.wmf_tableOfContentsSelectionIndicatorColor()
```

Comment:
The selection style is none… 

Also this isn't a UIKit provided view, `selectedSectionIndicator` is a view that I added. So it doesn't pick up on any of the selection style stuff.